examples/service-monitor.yaml

```diff
@@ -0,0 +1,22 @@
+---
```

Comment:
we need to update the document with full setup details on  how users can set up  Prometheus and use it with ceph-csi

note: if not possible in this PR, we can do it in the next PR also. if you want to do it in other PR create an issue so that we won't miss it.

some people are using ceph-csi without rook.

Reply:
I think a doc on how to setup prometheus is out of the scope of this repo. I will point to the promethues operator documentation on how to use that. https://coreos.com/operators/prometheus/docs/latest/user-guides/getting-started.html

Reply:
sound good.